akka-docs/src/main/paradox/general/message-delivery-reliability.md

```diff
@@ -177,8 +177,8 @@ particular:
 >
 > Parent actor `P` might receive the two events either in order `M`, `F` or `F`, `M`
 
-The reason for this is that internal system messages has their own mailboxes therefore the ordering of enqueue calls of
-a user and system message cannot guarantee the ordering of their dequeue times.
+The reason for this is that internal system messages have their own mailboxes therefore the ordering of enqueue calls of
+a user and system messages cannot guarantee the ordering of their dequeue times.
```

Comment:
'a' needs to be dropped here as well, I think, right?

Reply:
If `a` is dropped, it would be changed to `users`. I think both are correct.

Reply:
I don't think that's accurate here, as "user and system messages" here is short for "user messages and system messages".

Perhaps this also means we should simplify this sentence. Perhaps we should split it out into multiple sentences?

Reply:
Oh, thank for the correction! How about just a little verbose for the document? I use `user messages and system messages`.